### ISSUE TITLE
Report "Total size" in diff in units easy to perceive by humans

### ISSUE THREAD

**Issue by yarikoptic** [digint/btrbk#345]
loosely related to #280 as that one should do the same in its output.

ATM it looks like
```
lena# btrbk diff $PWD/.snapshots/home.2020102{1,2} | tail
WARNING: unparsed flags: PREALLOC
WARNING: unparsed flags: PREALLOC
WARNING: unparsed flags: PREALLOC
WARNING: unparsed flags: PREALLOC
WARNING: unparsed flags: PREALLOC
WARNING: unparsed flags: PREALLOC
WARNING: unparsed flags: PREALLOC
WARNING: unparsed flags: PREALLOC
WARNING: unparsed flags: PREALLOC
WARNING: unparsed flags: PREALLOC
WARNING: unparsed flags: PREALLOC
WARNING: unparsed flags: PREALLOC
WARNING: unparsed flags: PREALLOC
WARNING: unparsed flags: PREALLOC
WARNING: unparsed flags: PREALLOC
WARNING: unparsed flags: PREALLOC
WARNING: unparsed flags: PREALLOC
WARNING: unparsed flags: PREALLOC
WARNING: unparsed flags: PREALLOC
+.i    1           41  yoh/proj/repronim/reproman/.git/refs/heads/master
+.i    1           41  yoh/proj/repronim/reproman/.git/refs/pull/origin/551/head
+.i    1           41  yoh/proj/repronim/reproman/.git/refs/remotes/origin/master
+.i    1           41  yoh/proj/repronim/reproman/.git/refs/tags/v0.3.0
+..    1     16977920  yoh/shared/git-annex-windows-installer-20201021-magic.zip
+..    1     16982016  yoh/shared/git-annex-windows-installer-20201021-magic2016.zip
+..    1       307200  yoh/shared/msvcrt.dll.zip
+.i    1           52  yoh/tmp/mutt_source.txt

Total size: 102662224593 bytes
```
to me (notice the warnings, Debian with btrbk 0.29.1-1~nd100+1 perl 5.30.0-10), and I have no clue how many bytes it is besides that it is "many".  I would have appreciated to see `102.7 GB` instead.

**Comment by digint** [comment]
Implemented in 4866ec5cb082b39c61befb4305eb72cd80163c39, 528a152d6db6e0c63df0c549c80ba4e23c2fa19a

You can change the output by passing `--kbytes`, `--mbytes`, ..., as well as changing the table format using `--format=table|long|raw`.

**Comment by digint** [comment]
fyi I rebased the [extents-diff-v2 branch](https://github.com/digint/btrbk/tree/extents-diff-v2), which is able to give you much more accurate results. See `extents-diff` in [btrbk.1](https://github.com/digint/btrbk/blob/extents-diff-v2/doc/btrbk.1.asciidoc#informative-commands):

```
cd /tmp/
wget https://raw.githubusercontent.com/digint/btrbk/extents-diff-v2/btrbk
chmod +x btrbk
mkdir /tmp/btrbk.cache 

./btrbk --override=cache_dir=/tmp/btrbk.cache extents-diff /backup/data.* exclusive /backup/data.2010*
```

I'm still hesitating to merge it (as it's quite some code and btrbk is pretty big already), but I think it can prove quite useful

**Comment by digint** [comment]
Included in [v0.31.0](/digint/btrbk/releases/tag/v0.31.0)
